### PR TITLE
fix: shellcheck is failing

### LINKED ISSUE #978: fix shell check issue

### PR DESCRIPTION
**Pull request by litaocdl**
Closes #978 
Signed-off-by: Tao Li <tao.li@enterprisedb.com>

### CODE REVIEW

**Comment by litaocdl** [comment]
shell check pass https://github.com/cloudnative-pg/cloudnative-pg/actions/runs/3433841467

File: hack/e2e/run-e2e.sh [hunk 1]
```diff
@@ -119,7 +119,7 @@ mkdir -p "${ROOT_DIR}/tests/e2e/out"
 # would create CPUs-1 nodes and saturate the testing server
 RC_GINKGO2=0
 LABEL_FILTERS="!(upgrade)"
-if [ ! -z "${FEATURE_TYPE+x}" ]; then
+if [ -n "${FEATURE_TYPE+x}" ]; then
```
Comment (mnencia) [inline hack/e2e/run-e2e.sh:122]: Could you please use


```suggestion
if [ "${FEATURE_TYPE:-}" ]; then
```

**Review by mnencia — APPROVED** [review]

**Review by hh24k — APPROVED** [review]

**Pull request merged**